README.md

```diff
@@ -38,31 +38,38 @@ In order to use this library you can either use Mixins or have your models inher
    from django_multitenant.fields import *
    from django_multitenant.models import *
    ```
-1. All models should inherit the TenantModel class.
+2. All models should inherit the TenantModel class.
    `Ex: class Product(TenantModel):`
-1. Define a static variable named tenant_id and specify the tenant column using this variable.
-   `Ex: tenant_id='store_id'`
-1. All foreign keys to TenantModel subclasses should use TenantForeignKey in place of
+3. Define a static variable named tenant_id and specify the tenant column using this variable.You can define tenant_id in three ways. Any of them is acceptavle 
+   * Using TenantMeta.tenant_field_name variable
+   * Using TenantMeta.tenant_id variable
+   * Using template_id field
+> **Warning**
+> Using  template_id field directly in the class is not suggested since it may cause collision if class has a field named with 'tenant'
+4. All foreign keys to TenantModel subclasses should use TenantForeignKey in place of
    models.ForeignKey
-1. A sample model implementing the above 2 steps:
+5. A sample model implementing the above 2 steps:
   ```python
```

Comment:
One space too few in indentation. Fix indentation so the code block is becomes a child of the list item.

Reply:
This problem happens again later as well:

* under "A sample model implementing the above 3 steps"
* under "Creating foreign keys between tenant related models"
* under "Model.save() injects tenant_id for tenant inherited models"

Reply:
Fixed